content/en/real_user_monitoring/browser/tracking_user_actions.md

```diff
@@ -65,11 +75,11 @@ parents) that will be used to name the action. For example:
   Enter a valid email address
 </div>
 ```
-## Custom user actions
+## Custom actions
 
-Custom user actions are user actions declared and sent manually by using the `addAction` API. They can send information relative to an event occurring during a user journey (for example, a custom timing or customer cart information).
+Custom actions are user actions declared and sent manually by using the `addAction` API. They can send information relative to an event occurring during a user journey (for example, a customer cart information).
 
-Once RUM is initialized, generate user actions when you want to monitor specific interactions on your application pages or measure custom timings with the `addAction(name: string, context: Context)` API call. In the following examples, the RUM SDK collects the number of items within a cart, what they are, and how much the cart is worth overall.
+Once RUM is initialized, generate custom actions when you want to monitor specific interactions on your application pages with the `addAction(name: string, context: Context)` API call. In the following examples, the RUM SDK collects the number of items within a cart, what they are, and how much the cart is worth overall. The information is sent to Datadog when the user goes to the checkout page.
```

Comment:
> The information is sent to Datadog when the user goes to the checkout

I feel that it is a bit unclear, since we don't mention a checkout page beforehand. Maybe we could reword this a bit like:

> The following example illustrates how to collect and send custom information to Datadog when the user checkout their cart.